test/routes/sessions.test.js

```diff
@@ -12,12 +12,16 @@ const envSecret = Array(65).join('0');
 const authSecret = Array(65).join('1');
 const twoFactorAuthenticationSecret = '00000000000000000000';
 
-function setupApp() {
+let app;
+let urlService;
+let nockObj;
```

Comment:
The reason of the `jest/no-hooks` lint rule existence is to prevent potential shared states across tests cases as it looks to be done here.
(as mentioned here: https://github.com/jest-community/eslint-plugin-jest/blob/master/docs/rules/no-hooks.md)
Another reason to follow the rule.

Reply:
Those tests share the same app instance, and it seems mandatory.
This is related to the fact than `Liana.init` is stateful: we can't have multiple isolated forestApp instances.
I consider showing explicitly the stateful behavior of app a clearer choice.
(Furthermore, I add a beforeEach because it is explicit and allow async code)